batchUploadResourceCollections.sh

```diff
@@ -20,7 +20,7 @@ for dir in ./*; do
     fi
 done
 
-
+cd ../../..
```

Comment:
Since this script is run and not sourced (e.g. with `source` or `.`), it runs in its own subshell and any `cd` commands it runs will not affect the present working directory of the user's shell